templates/hound.yml

```diff
@@ -6,12 +6,12 @@ haml:
   # config_file: .haml-style.yml
   enabled: true
 javascript:
-  # config_file: .javascript-style.json
+  config_file: .jscsrc
```

Comment:
The `javascript` key is an alias for the JSHint linter, so giving it a config file for JSCS would probably cause Hound to fail. Typically people use both JSHint and JSCS. For that, hound.yml would look like:

```yaml
javascript:
  enabled: true
  config_file: .jshintrc

jscs:
  enabled: true
  config_file: .jscsrc
```

However, JSCS is [merging] (or has merged?) with ESLint, and Hound won't support JSCS for much longer because of that. It seems to me like ESLint is becoming the dominant JavaScript linter, so maybe it would make sense to configure that in hound.yml in this PR.

[merging]: http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint

```yaml
javascript:
  enabled: false

eslint:
  enabled: true
```